src/Web/Instances.hs

```diff
@@ -1,27 +0,0 @@
-{- |
```

Comment:
Can you tell me what happened here? Did the instance definition move somewhere else?

Reply:
This file (and instances) are not used anymore